Makefile

```diff
@@ -224,6 +224,16 @@ copy-branding:
 	mkdir -p $(current_dir)/translations
 	cp -a $(current_dir)/branding $(current_dir)/translations/
 
+.PHONY: configure-suma-branding-webui
+configure-suma-branding-webui:
```

Comment:
Future improvement could be taking this value from a simple config file that can be parsed with bash :-)